src/04_printing.py

```diff
@@ -11,7 +11,11 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print("x is %i, y is %.2f, z is "'"%s"'" " % (x, y, z))
 
 # Use the 'format' string method to print the same thing
+string = "x is {}, y is {}, z is "'"{}"'" "
```

Comment:
you can use a version of the  `.2f` syntax from the `printf %` method in `.format` and `f`string methods, ask if you don't figure it out